src/index.ts

```diff
@@ -187,6 +259,7 @@ const cloudCheck = async (req: express.Request, res: express.Response) => {
 
     // route definition
     app.post("/", cloudCheck);
+    app.post("/ioTest", ioTest);
```

Comment:
I would call this endpoint "run" instead of ioTest since this is basically a generic code runner. It's also best to avoid mixing uppercase and lowercase when defining endpoints in an API and stick to lowercase only.